lib/syskit/log/dsl.rb

```diff
@@ -780,6 +783,44 @@ def to_html
             def raw_html(html)
                 RawHTML.new(html)
             end
+
+            # Export streams from the dataset into a new multi-stream log file
+            #
+            # @param [Array] streams an array of objects that can be converted to
+            #   samples using {#samples_of}
+            # @yield a {DSL::AlignmentBuilder} object used to describe the streams to be
+            #   saved
+            def export_to_single_file(path, *streams)
+                log_file = Pocolog::Logfiles.create(path.to_s)
+                return if streams.empty?
```

Comment:
```suggestion
                return if streams.empty?
                
                log_file = Pocolog::Logfiles.create(path.to_s)
```

Reply:
I actually don't agree. If you define no streams, I am still expecting it to create an empty log file.